docs/platforms/apple/common/troubleshooting/index.mdx

```diff
@@ -115,3 +115,6 @@ SentrySDK.start { options in
     options.enableSwizzling = NO;
 }];
 ```
+## error: invalid version number in '-target arm64-apple-ios9999'
+
+If you see the error `error: invalid version number in '-target arm64-apple-ios9999'` when building your app, it's because you're using Xcode 15.4 or above with Sentry version lower than 8.26.0. To fix this, update your Sentry version to at least 8.26.0.
```

Comment:
```suggestion
## error: invalid version number in '-target arm64-apple-ios9999'

If you see the error `error: invalid version number in '-target arm64-apple-ios9999'` when building your app, it's because you're using Xcode 15.4 or above with a Sentry SDK version that's lower than 8.26.0. To fix this, update your SDK to version 8.26.0 or above.
```